### DISCUSSION TITLE
Replicate different version (1.7.4 to 2.7.0)

### DISCUSSION THREAD

**Discussion by SiamPiApinant** [goharbor/harbor#18330] [Q&A]
Hi ,
Now I have use harbor 1.7.4 for old project and I have init new harbor (2.7.0) for new project. I try set replication rule to sync image but have an errors . Question is I should upgrade old harbor before sync or manual pull image and push to new harbor ?

Or some one have any idea to solve this .

Thank you in advance

**Comment by chlins** [comment]
I think you could back up the 1.7.4 harbor and then upgrade it to 2.7.0 then the same version replication should be supported.
